README.md

```diff
@@ -0,0 +1,73 @@
+# Installation
+
+To reproduce the website in your local machine follow these steps.
+
+## Clone repo and submodules
+
+    git clone --recurse-submodules git@github.com:PyLadiesBerlin/website.git
+
+For git < 2.11 use:
+
+    git clone --recursive git@github.com:PyLadiesBerlin/website.git
+
+## Update submodules
+
+    git submodule update
+
+## Setup an environment
+
+### Using python
+
+    python3 -m venv .env  # use python if it is a python > 3
+    source .env/bin/activate
+
+#### Install requirements
+
+    pip install -r requirements.txt
+
+### Using conda or mamba
+
+With conda:
+
+    conda create -n pyladies
+    conda install -c conda-forge pelican markdown beautifulsoup4
+
+Or mamba:
+
+    conda create -n pyladies
+    mamba install -c conda-forge pelican markdown beautifulsoup4
+
+## Build and preview content
+
+Access the `content` directory
+
+    cd content
```

Comment:
I think the reason it worked with cd content is that it didn't find the settings that mention the needed plugins and theme. In this case you will not get the correct webite look, that is like here https://pyladiesberlin.github.io/

Is that right?

Reply:
yes, i'm almost certain that's what I meant